src/LibEventLoop.php

```diff
@@ -31,6 +31,10 @@ class LibEventLoop implements LoopInterface
 
     public function __construct()
     {
+        if (version_compare(PHP_VERSION, '7.0.0') >= 0) {
+            trigger_error('The libevent extension has not yet been updated for PHP 7, it is recommended you use a different loop', E_USER_WARNING);
+        }
```

Comment:
I'm not a big fan of this tbqh :-1: 

With the above check in place we do not automatically create the `LibEventLoop` from our `Factory`. Afaict this is what most people use, so this should resolve this for the most part?

This means that if this constructor is invoked, it's explicitly been called. IMHO this is already advanced usage and we should not trigger a warning in this case.

Otherwise, how could a user possibly ignore this warning? Using `@`-suppression would result in *zero* output if the extension can not be loaded all.

Also, what happens when the extension will be updated to support PHP 7 again? We can't really detect this until we provide an update, in which case the above warning would be superfluous.

Reply:
If the user has upgraded to PHP7 and somehow kept the extension installed, from the reports I've received, it's going to segfault. The trigger_error only issues a warning, the object will still be instantiated an be able to be used. 

> Also, what happens when the extension will be updated to support PHP 7 again?

It looks like that extension isn't going to be...but if it does we'll have to quickly update this to support it. Even if this happens the compare only issues a warning.

Reply:
If the consensus is still a :-1: on this I'm good to remove this check in order to get the updated Factory order into the next release.

Reply:
I'm not sure we had a consensus here, it was basically just my point of view :-) But if this error is removed, I'm all for getting this in! :+1: